### PR TITLE
Home page redesign colors buttons

### PR DESCRIPTION
**Pull request by mcbouslog**
WIP for edits to colors, buttons, minor content edits and other misc styling.

### CODE REVIEW

File: css/home-page-redesign.styl [hunk 1]
```diff
@@ -39,35 +41,45 @@ GRAY = #EFF2F5
...
+    &:focus
+      background: #FFFFFF
+      color: DARK-BLUE-GREEN
 
   .home-link-down
     width: 100%
     text-align: center
     border: none
-    background-color: TEAL
+    background: rgba(0, 0, 0, 0.0)
```
Comment (parrish) [inline css/home-page-redesign.styl]: Do you mean `background: none`?

File: css/home-page-redesign.styl [hunk 2]
```diff
@@ -76,13 +88,41 @@ GRAY = #EFF2F5
       z-index: -1
       width: 100%
 
-    @media (max-width: 700px)
-      padding-top: 8vw
+      @media (max-width: 700px)
+        width: auto
+
+    .home-intro-content
```
Comment (parrish) [inline css/home-page-redesign.styl]: This just duplicates `section`.  I think I'm missing the intent

File: app/pages/home-not-logged-in.cjsx [hunk 3]
```diff
@@ -32,30 +32,31 @@ module.exports = React.createClass
             <source type="video/mp4" src="./assets/home-video.mp4"></source>
           </video>
         }
-
-        <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
-        <h2 className="lighter">is people powered research</h2>
-        <h2>
-          <Link to="/about" className="standard-button">Learn More</Link>
-        </h2>
+        <div className="home-intro-content">
```
Comment (parrish) [inline app/pages/home-not-logged-in.cjsx]: This extra markup isn't really necessary.

File: app/pages/home-not-logged-in.cjsx [hunk 4]
```diff
@@ -32,30 +32,31 @@ module.exports = React.createClass
...
+          <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
+          <h2 className="lighter">is people powered-research</h2>
+          <div className="home-intro-buttons">
+            <button className="button standard-button" onClick={@scrollDown}>Learn More</button>
+            <Link to="/projects" className="button intro-button">Get Started</Link>
+          </div>
+        </div>
+        <button className="home-link-down" onClick={@scrollDown}>
```
Comment (parrish) [inline app/pages/home-not-logged-in.cjsx:38]: Moving this into the section element is causing some weird partial underneath the section.

File: app/pages/home-not-logged-in.cjsx [hunk 5]
```diff
@@ -32,30 +32,31 @@ module.exports = React.createClass
...
-          The Zooniverse enables everyone to take part in real cutting edge
+          The Zooniverse enables everyone to take part in real cutting-edge
           research in many fields across the sciences, humanities, and more.
           The Zooniverse creates opportunities for you to unlock answers and
           contribute to real discoveries.
         </p>
 
         <h2>
-          <Link to="/projects" className="standard-button">Get started</Link>
+          <Link to="/projects" className="button discover-button">Choose a Project</Link>
```
Comment (parrish) [inline app/pages/home-not-logged-in.cjsx]: You can drop the extra class name and just specify the css as `.home-discover .standard-button` or something similar

**Comment by parrish** [comment]
Pinging @jwbmartin too.

I like the behavior of the learn more / get started buttons now.

I'm really bothered by the button style changes.

"Get started" needs to be the call to action, but it's currently de-emphasized.  The background-color blends in with the video and the box-shadow isn't helping.  The shadow is also creating some alignment issues with the other button (different size and baseline)

It feels spastic when the background-color and text color changes dramatically on hover/click/focus.

I also think it's best if we just drop the down arrow.  It just feels unnecessary.

**Comment by jwbmartin** [comment]
I think the down arrow is still pretty standard though but it could be something lighter and more subtle (example: http://www.bukwild.com/) so maybe taking down the stroke weight will help. 

I'm still playing around a bit with button styling so, hopefully, we'll have a more solid idea soon.

**Comment by parrish** [comment]
That site is pretty funky because I'm scrolling down before I see a button to click to scroll down.  It's also present for a few sections and then disappears.  I just started thinking it's silly to assist users with scrolling.

Happy to chat about anything if you like.

**Comment by jwbmartin** [comment]
Hm I don't have that experience with that site. I'm not dedicated to the arrow, it's just something I've seen very often on sites which could mean it's just a fad or indicative of something we should consider. Statistically, users will scroll down no matter what so I'm not too terribly concerned.

**Comment by mcbouslog** [comment]
@parrish - notes above made sense, I think I've properly updated but let me know what you think. there's a chance we might tweak as we continue to put the pieces together, but hopefully this is a good small step.

File: app/pages/home-not-logged-in.cjsx [hunk 6]
```diff
@@ -32,30 +32,29 @@ module.exports = React.createClass
             <source type="video/mp4" src="./assets/home-video.mp4"></source>
           </video>
         }
-
         <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
-        <h2 className="lighter">is people powered research</h2>
-        <h2>
-          <Link to="/about" className="standard-button">Learn More</Link>
-        </h2>
+        <h2 className="lighter">is people powered-research</h2>
```
Comment (parrish) [inline app/pages/home-not-logged-in.cjsx]: "powered-research" shouldn't be hyphenated.  Alex mentioned hyphenating "people powered" , but that's optional grammar.  (Personally, I wouldn't)

File: app/pages/home-not-logged-in.cjsx [hunk 7]
```diff
@@ -32,30 +32,29 @@ module.exports = React.createClass
...
-      <button className="home-link-down" onClick={@scrollDown}>
-        <i className="fa fa-angle-down" onClick={@scrollDown} />
-      </button>
-
       <section className="home-discover" ref="discover">
         <h1>Discover, teach, and learn</h1>
 
         <p>
-          The Zooniverse enables everyone to take part in real cutting edge
+          The Zooniverse enables everyone to take part in real cutting-edge
```
Comment (parrish) [inline app/pages/home-not-logged-in.cjsx]: Ditto.  Hyphenation isn't necessary, but is optional.

File: css/home-page-redesign.styl [hunk 8]
```diff
@@ -91,21 +133,34 @@ GRAY = #EFF2F5
...
-    .standard-button
-      border: 4px solid NAVY
+      @media (max-width: 900px)
+        margin-top: 20px
 
       &:hover
-        background: rgba(0, 0, 0, 0.1)
+      &:active,
+      &:focus
+        border-color: GRAY
```
Comment (parrish) [inline css/home-page-redesign.styl]: By matching the border to the background, you're basically removing it.  It makes the button "bouncy."  I'd remove this and just leave it as `.discover-button{ border: 2px solid DARK-BLUE; }`.

File: css/home-page-redesign.styl [hunk 9]
```diff
@@ -91,21 +133,34 @@ GRAY = #EFF2F5
...
+        background: rgba(12, 72, 129, 0.9)
```
Comment (parrish) [inline css/home-page-redesign.styl]: Fun fact, you can specify this as `background: rgba(DARK-BLUE, 0.9)`

File: css/home-page-redesign.styl [hunk 10]
```diff
@@ -40,34 +42,45 @@ GRAY = #EFF2F5
...
+      color: #FFFFFF
+
+    &:hover,
+    &:active,
+    &:focus
+      background: #FFFFFF
+      color: DARK-BLUE-GREEN
 
   .home-link-down
+    position: absolute
```
Comment (parrish) [inline css/home-page-redesign.styl:75]: This is super broken in firefox now:
link-down

Specifying `left: 0` will fix it.

File: css/home-page-redesign.styl [hunk 11]
```diff
@@ -40,34 +42,45 @@ GRAY = #EFF2F5
...
 
-  .home-intro
-    color: #FFFFFF
+    @media (max-width: 900px)
+      display: none
 
-    h1
-      color: #FFFFFF
+  .home-intro
+    background-color: rgba(91, 189, 193, 0.3)
```
Comment (parrish) [inline css/home-page-redesign.styl]: We really don't want some sort of colored overlay do we?  Probably should remove this.

**Comment by mcbouslog** [comment]
sorry for the delay! random NfN stuff last week but will be focusing on home-page this week. thanks for your patience and back-and-forth @parrish and @jwbmartin!

**Pull request closed without merging**